### PR TITLE
Make the event and blog post edit screens responsive

### PR DESCRIPTION
**Pull request by danrschlosser**
Review: @eunicekokor @RaymondXu

Events:
screen shot 2015-04-03 at 1 12 46 am

Blog posts:
screen shot 2015-04-03 at 1 12 54 am

### CODE REVIEW

**Comment by danrschlosser** [comment]
Adding @natebrennand in case you want to take a look too

File: app/templates/admin/posts/edit.html [hunk 1]
```diff
@@ -31,14 +31,16 @@
...
-{{ macros.toggle("Publish", "Unpublish")}}
-{% else %}
-{{ macros.toggle("Publish", "Unpublish", start_on=False)}}
-{% endif %}
-{% endif %}
+<span class="edit-post-barright">
+    <a class="text-button" href="#save">Save Draft</a>
+    {% if user.can("publish") %}
+    {% if post and post.published %}
+    {{ macros.toggle("Publish", "Unpublish")}}
```
Comment (natebrennand) [inline app/templates/admin/posts/edit.html]: can you indent these blocks so it's more obvious that there's nested logic?

**Comment by natebrennand** [comment]
I can't really comment on the `scss` changes but LGTM after the indentation

**Comment by danrschlosser** [comment]
ping @RaymondXu @eunicekokor I'd love it if one of you would give this a read-through

**Comment by RaymondXu** [comment]
:+1: looks good

**Pull request merged**